worlds/ffmq/Client.py

```diff
@@ -47,6 +47,18 @@ def get_flag(data, flag):
     bit = int(0x80 / (2 ** (flag % 8)))
     return (data[byte] & bit) > 0
 
+def validate_read_state(data1, data2):
+    validation_array = bytes([0x01, 0x46, 0x46, 0x4D, 0x51, 0x52])
+
+    if data1 is None or data2 is None:
+        return False
+    else:
+        for i in range(6):
+            if data1[i] != validation_array[i] or data2[i] != validation_array[i]:
+                return False;
```

Comment:
Since there's a return, you don't need the `else`
```suggestion
    for i in range(6):
        if data1[i] != validation_array[i] or data2[i] != validation_array[i]:
            return False;
```